File: TransCoder_tokenized_test_set_functions/python/GROUP_MULTIPLE_OCCURRENCE_OF_ARRAY_ELEMENTS_ORDERED_BY_FIRST_OCCURRENCE.py
def groupElements ( arr, n ) :
    visited = [ False ] * n
    for i in range ( 0, n ) :
        visited [ i ] = False
    for i in range ( 0, n ) :
        if ( visited [ i ] == False ) :
            print ( arr [ i ], end = " " )
            for j in range ( i + 1, n ) :
                if ( arr [ i ] == arr [ j ] ) :
                    print ( arr [ i ], end = " " )
                    visited [ j ] = True
    

File: TransCoder_tokenized_test_set_functions/java/GROUP_MULTIPLE_OCCURRENCE_OF_ARRAY_ELEMENTS_ORDERED_BY_FIRST_OCCURRENCE.java
static void groupElements ( int arr [ ], int n ) {
  boolean visited [ ] = new boolean [ n ];
  for ( int i = 0;
  i < n;
  i ++ ) {
    visited [ i ] = false;
  }
  for ( int i = 0;
  i < n;
  i ++ ) {
    if ( ! visited [ i ] ) {
      System . out . print ( arr [ i ] + " " );
      for ( int j = i + 1;
      j < n;
      j ++ ) {
        if ( arr [ i ] == arr [ j ] ) {
          System . out . print ( arr [ i ] + " " );
          visited [ j ] = true;
        }
      }
    }
  }
}